examples/aws/example-build/envs.sh

```diff
@@ -27,4 +27,7 @@ export TF_VAR_root_dns_zone_id=<aws route53 zone id>
 export TF_VAR_subnet_bastion_0=<bastion subnet cidr 1>
 export TF_VAR_subnet_bastion_1=<bastion subnet cidr 2>
 
+# Logging bucket expiration days
+export TF_VAR_expiration_days=<expiration days>
+
```

Comment:
For the real clusters, should I set this var in conveyor? how can I get it from installations?

Reply:
Just use meaningful default in variables.tf. No need to set this everywhere.

Reply:
Do you have this plan now? All variables available [here](https://github.com/giantswarm/giantnetes-terraform/blob/master/platforms/aws/giantnetes/variables.tf) and there are bunch of them. 
I want to avoid having "monster" size `envs.sh`. If you will need this for `viking` you can do this, but i don't think that for every new installation we should decide on logging retention period

Reply:
Ok so, for now, we can avoid variables, and set up as default the viking desired value. In the future if there is other client wants a different policy then we can implement. WDYT?

Reply:
let's have a call i'll describe my point :)

i want to remove it from `envs.sh` not from variables.tf